packages/ethereum/deploy/06_HoprNetworkRegistryProxy.ts

```diff
@@ -0,0 +1,38 @@
+import type { HardhatRuntimeEnvironment } from 'hardhat/types'
+
+const PROTOCOL_CONFIG = require('../../core/protocol-config.json')
+const MIN_STAKE = 0
+
+// Deploy directly a HoprNetworkRegistry contract, using hardcoded staking contract.
+const main = async function (hre: HardhatRuntimeEnvironment) {
+  const { ethers, deployments, getNamedAccounts, network, environment } = hre
+
+  const environmentConfig = PROTOCOL_CONFIG.environments[environment]
+  const deployer = await getNamedAccounts().then((o) => ethers.getSigner(o.deployer))
+
+  const adminAddress =
+    network.name == 'hardhat' ? deployer.address : environmentConfig['network_registry_admin_address']
+
+  // deploy different contracts depending on the environment
+  const registryProxy =
+    environmentConfig['network_id'] == 'xdai'
```

Comment:
I guess here we should be looking whether this is `production` && `xdai` || `development` && `goerli` ?
I remember we used to have tags that would signify whether a network was `production`, `development` or `testing` but I don't see that.
Or should we instead use the `live` entry in protocol config?

Reply:
lets park this now as am working on this in https://github.com/hoprnet/hoprnet/issues/3687